notebooks/amused-lightweight-text-to-image/amused-lightweight-text-to-image.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>could you please add link on inception score explanation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Added a link</p>